src/js/tracks/text-track-display.js

```diff
@@ -103,7 +121,20 @@ class TextTrackDisplay extends Component {
       cues.push(track['activeCues'][i]);
     }
 
-    window['WebVTT']['processCues'](window, track['activeCues'], this.el_);
+    if (track['kind'] === 'descriptions') {
+
+      if ((overrides.descriptionsPlayback) && (overrides.descriptionsPlayback === 'screenReaderOnly')) {
```

Comment:
Currently, if you enable both descriptions and captions at the same time, they would overlap each other.